centaur/src/main/resources/standardTestCases/read_file_from_bucket_with_requester_pays/read_file_from_bucket_with_requester_pays.wdl

```diff
@@ -0,0 +1,21 @@
+task concat_task{
+    Array[String] array
+
+    command {
+        echo $${sep = ' ' array} > concat
```

Comment:
is the double `$` deliberate?

Reply:
No. I just saw in some other WDL so added it. Removed it now.